templates/init/functions/python/requirements.txt

```diff
@@ -1 +1,2 @@
-firebase_functions~=0.1.0
\ No newline at end of file
+firebase_functions~=0.1.0
+firebase-admin~=6.1.0
```

Comment:
@taeold per our chat, it sounds like this line isn't actually necessary. Is there a risk of weird behavior if the `firebase-admin` version here gets out of sync with the `firebase-admin` version expected by `firebase_functions`?